### PR TITLE
fix: ChatGPT connector

### PR DESCRIPTION
**Pull request by Kahtaf**
Also includes prettierrc file for consistent front-end formatting

### CODE REVIEW

File: selfie-ui/.prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (tnunamak) [inline selfie-ui/.prettierrc.json:1]: Shall we also install prettier and integrate it with ESLint?

**Review by tnunamak — APPROVED** [review]

File: selfie-ui/.prettierrc.json (hunk 1 above)
Comment (Kahtaf) [reply selfie-ui/.prettierrc.json:1]: Agreed, let me do that in a separate PR

Comment (Duonget123) [reply selfie-ui/.prettierrc.json:1]: https://docs.github.com/en/enterprise-cloud@latest/admin/managing-iam

**Review by bbloo21 — COMMENTED** [review]
Good

**Pull request merged**